src/sidebar/test/markdown-commands-test.js

```diff
@@ -105,55 +104,47 @@ describe('markdown commands', function() {
         input: '<sel></sel>',
         output: '> <sel></sel>',
       },
-    ];
-
-    unroll(
-      '#tag',
-      function(fixture) {
+    ].forEach(fixture => {
+      it(`${fixture.tag}`, () => {
```

Comment:
``it(`${fixture.tag}`,`` can be simplified to just `it(fixture.tag,`

Reply:
lol @ myself for this one :)